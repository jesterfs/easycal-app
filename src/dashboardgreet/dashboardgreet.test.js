import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import DashboardGreet from './dashboardgreet'
import renderer from 'react-test-renderer';

describe('DashboardGreet component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <DashboardGreet />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <DashboardGreet />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});