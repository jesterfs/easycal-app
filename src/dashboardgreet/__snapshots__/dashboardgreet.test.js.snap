// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardGreet component renders the UI as expected 1`] = `
<div
  className="DashboardGreet, greetgroup"
>
  <div
    className="item"
  />
  <div
    className="item"
  >
    <form
      id="selectCalendarForm"
      onSubmit={[Function]}
    >
      <select
        htmlFor="selectCalendarForm"
        name="calendar"
      >
        <option>
          select a calendar
        </option>
      </select>
      <input
        className="addbtn"
        htmlFor="selectCalendarForm"
        type="submit"
      />
    </form>
  </div>
  <div
    className="item"
  >
    <h3 />
  </div>
  <div
    className="addButtons"
  >
    <div
      className="item, dashboardbtn"
    >
      <a
        href="/addcalendar"
        onClick={[Function]}
      >
        <button
          className="addbtn"
        >
          Add A Calendar
        </button>
      </a>
    </div>
    <div
      className="item, dashboardbtn"
    >
      <a
        href="/addevent"
        onClick={[Function]}
      >
        <button
          className="addbtn"
        >
          Add An Event
        </button>
      </a>
    </div>
    <div
      className="item, dashboardbtn"
    >
      <a
        href="/addmember"
        onClick={[Function]}
      >
        <button
          className="addbtn"
        >
          Add A Team Member
        </button>
      </a>
    </div>
  </div>
</div>
`;
